clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts

```diff
@@ -233,14 +233,12 @@ describe("Config wizard with plus settings", () => {
       cy.getByTestId("add-system-form");
     });
 
-    // TODO: Update Cypress test to reflect the nav bar 2.0
-    it.skip("Resets the flow when it is completed", () => {
+    it("Resets the flow when it is completed", () => {
       goToDataFlowScanner();
       cy.wait("@putScanResults");
       cy.getByTestId("scan-results");
       cy.getByTestId("register-btn").click();
-      cy.getByTestId("nav-link-Add Systems").click();
-      cy.getByTestId("setup");
+      cy.getByTestId("nav-link-add-systems").click();
```

Comment:
I think there needs to be one more statement here to do what the test says it does (that it resets). maybe a `cy.getByTestId("add-system-form");`?